fec-icons/icon-template.scss

```diff
@@ -0,0 +1,31 @@
+// NOTE: Do note edit _icon-variables.scss directly
+// It is generated by Gulp from fec-icons/icon-template.scss
```

Comment:
I guess you'd back in the file then?

So this file will be generated and have a bunch of base64 encoded files based on svg icons. My concern would be that due to small version differences in npm packages on different people's computers, there could be slight variations in how the svgs are compiled which will lead to variations in the base64 encoded strings leading to differences in the checked in file which could cause merge conflicts (we ran into this exact issue with a js map file in WDS). This is all speculation, so might not end up being a concern.